docs/resources/setting_mgmt.md

```diff
@@ -0,0 +1,27 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "unifi_setting_mgmt Resource - terraform-provider-unifi"
+subcategory: ""
+description: |-
+  unifi_setting_mgmt manages settings for a unifi site.
+---
+
+# unifi_setting_mgmt (Resource)
+
+`unifi_setting_mgmt` manages settings for a unifi site.
+
+
```

Comment:
You can add an example by creating a file at `examples/resource/unifi_setting_mgmt/resource.tf` and then regenerating docs.